### PR TITLE
MO screen orientation

### PR DESCRIPTION
**Pull request by juliennassar**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Running your Dangerfile has Failed
  </td>
    </tr>
  </tbody>
</table>




## Danger has errored

Error: ReferenceError

```sh
ReferenceError: standardFile is not defined
    at Object.<anonymous> (dangerfile.js:28:29)
    at Runtime._execModule (/home/travis/build/bamlab/dev-standards/node_modules/jest-runtime/build/index.js:513:13)
    at Runtime.requireModule (/home/travis/build/bamlab/dev-standards/node_modules/jest-runtime/build/index.js:329:14)
    at /home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:133:33
    at ensureCleanDangerfile (/home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:173:5)
    at Object.<anonymous> (/home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:132:21)
    at step (/home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:32:23)
    at Object.next (/home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:13:53)
    at /home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:7:71
    at __awaiter (/home/travis/build/bamlab/dev-standards/node_modules/danger/distribution/runner/DangerfileRunner.js:3:12)
```


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: react-native/react/screen-orientation.mo.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+## Steps (~5 minutes)
+
+- for iOS, open XCode, in your project settings -> general -> deployment info check the devices orientations you want to allow on your app :
+
+ex : In this example I only allowed Portrait mode
+
+<img width="864" alt="screen shot 2017-10-22 at 13 06 22" src="https://user-images.githubusercontent.com/13121639/31861233-ef267ec4-b729-11e7-80d4-3eb8126dccc4.png">
+
+
+Then kill your packager and rebuild your app with `react-navite run-ios`
```
Comment (Almouro) [inline react-native/react/screen-orientation.mo.md:25]: you don't have to kill your packager

File: react-native/react/screen-orientation.mo.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+## Prerequisites (~ 2 minutes)
+- [ ] Have a React Native application [using yo generator](https://github.com/bamlab/generator-rn-toolbox)
+
+# Force a unique orientation for your app
+
+## Steps (~5 minutes)
+
+- for iOS, open XCode, in your project settings -> general -> deployment info check the devices orientations you want to allow on your app :
+
+ex : In this example I only allowed Portrait mode
```
Comment (Almouro) [inline react-native/react/screen-orientation.mo.md:20]: no space before `:`

File: react-native/react/screen-orientation.mo.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# [MO] Handle screen orientation (iOS & Android)
+## Owner: [Julien Nassar](https://github.com/juliennassar)
+## Sponsor: [Alexandre Moureaux](https://github.com/almouro)
+
+> **Note**: Please create an [issue](https://github.com/bamlab/dev-standards/issues/new) or even a Pull Request with your feedback, typo correction.
+
+## Context
+
+You may sometimes need to control the screen orientation of your app. You might want to fix your screen orientation to portrait mode only, or, depending on the page you are in, you might want to specify a screen orientation for some of your screens.
```
Comment (Almouro) [inline react-native/react/screen-orientation.mo.md:9]: do you have example of changing orientation for some screens?
I can think of UGC: changing orientation for full screen videos only

File: react-native/react/screen-orientation.mo.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      </activity>
+    </application>
+
+</manifest>
+```
+
+# Force a multiple orientation constraints (~5 minutes)
+
+Use the [`react-native-orientation`](https://github.com/yamill/react-native-orientation) package :
+`yarn add react-native-orientation``
```
Comment (Almouro) [inline react-native/react/screen-orientation.mo.md:52]: `` -> `

File: react-native/react/screen-orientation.mo.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+You have to link the package with `react-native link` to use it, then rerun your app `react-native run-ios` (rebuild native code)
+and follow documentation ;)
```
Comment (Almouro) [inline react-native/react/screen-orientation.mo.md:54]: can you give an example? For instance unlocking orientation for a single screen?

**Review by Almouro — CHANGES_REQUESTED** [review]

**Comment by tychota** [comment]
I pinged @juliennassar : goal merge this this week.

**Comment by tychota** [comment]
No Activity feel free to reopen. Archiving.

**Pull request closed without merging**